docs/pages/messages/invalid-env-prefix.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Invalid Environment Variable Prefix
+description: "invalid-env-prefix error page"
+---
+
+# Invalid Environment Variable Prefix
+
+When declaring environment variables in your `turbo.json`, you cannot prefix them with `$`. This
+was an old syntax for declaring a dependency on an environment variable that is no longer used.
+
+## Example
```

Comment:
Should we change this to "Why this error occurred"? There isn't really a why here, just an example @anthonyshew

Reply:
If the header moves above the paragraph, it will make sense that way!